controllers/humiocluster_controller.go

```diff
@@ -1922,6 +1928,64 @@ func (r *HumioClusterReconciler) ensureMismatchedPodsAreDeleted(ctx context.Cont
 
 	podsForDeletion := desiredLifecycleState.podsToBeReplaced
 
+	pdbSpec := hnp.GetPodDisruptionBudget()
+	r.Log.Info("Entering PDB enforcement check", "nodePool", hnp.GetNodePoolName(), "pdbSpec", pdbSpec, "pdbSpec.Enabled", pdbSpec != nil && pdbSpec.Enabled)
+	if pdbSpec != nil && pdbSpec.Enabled {
+		pdbName := fmt.Sprintf("%s-%s-pdb", hc.Name, hnp.GetNodePoolName())
+		pdb := &policyv1.PodDisruptionBudget{}
+
+		r.Log.Info("Fetching PDB",
+			"pdbName", pdbName,
+			"namespace", hnp.GetNamespace())
```

Comment:
You don't need to add the namespace to the logs yourself as we initialize the logger in the start of the `Reconcile()` function with the namespace: https://github.com/humio/humio-operator/blob/master/controllers/humiocluster_controller.go#L91-L96

With this, it means all logs using `r.Log` would include both the name of the `HumioCluster` resource as well as the namespace of where it is located.

Reply:
Thanks for the info.  Actually, I was unsure about the log message structure that and was going to ask about it.  I will keep it simple as `r.Log.Info("Fetching PDB",
			"pdbName", pdbName)`